talk/owt/sdk/base/customizedoutputaudiodevicemodule.cc

```diff
@@ -0,0 +1,109 @@
+// Copyright (C) <2023> Intel Corporation
```

Comment:
Can you explain why we need this, and why attaching an fake output device and then AddSink will not work?

Reply:
If we use FakeAudioDevice, recording is not available so no data will be sent to audio sink. This class makes recording available.